src/ipcache.cc

```diff
@@ -1226,7 +1226,7 @@ snmp_netIpFn(variable_list * Var, snint * ErrP)
 
     default:
         *ErrP = SNMP_ERR_NOSUCHNAME;
-        snmp_var_free(Answer);
+        /*snmp_var_free(Answer); */
```

Comment:
@Eusgor, since you closed this PR, I moved the suggested fix to #1097. If you would like, please let me know how to spell your name and/or email, and I will add the corresponding `Co-authored-by:...` footer to that dedicated PR. This is totally optional, of course. Thank you.